File: FbxSharpTests/SubDeformerTest.cs
using System;
using NUnit.Framework;
using FbxSharp;

namespace FbxSharpTests
{
    [TestFixture]
    public class SubDeformerTest : TestBase
    {
        [Test]
        public void SubDeformer_Create_HasNamespacePrefix()
        {
            // given:
            var obj = new FbxCluster("asdf");

            // then:
            Assert.AreEqual("SubDeformer::", obj.GetNameSpacePrefix());;
        }
    }
}

File: FbxCppTests/SubDeformerTest.cpp

#include "Tests.h"

using namespace std;

void SubDeformer_Create_HasNamespacePrefix()
{
    // given:
    FbxManager* manager = FbxManager::Create();
    FbxCluster* obj = FbxCluster::Create(manager, "asdf");

    // then:
    AssertEqual("SubDeformer::", obj->GetNameSpacePrefix());;
}

void SubDeformerTest::RegisterTestCases()
{
    AddTestCase(SubDeformer_Create_HasNamespacePrefix);
}

